### PR TITLE
This is the new update that I have so far

### PR DESCRIPTION
**Pull request by kaimanasse15**
### **Pull requests should follow a common structure for organizational purposes and quick readability.** 

# **Naming convention for PR titles**
< repo > _ < sprint > _ < issue > _ < pr or issue name >\
\
Example:\
belindas-closet-android_2_5_landing-page\
This would be for the belindas-closet-android repo, sprint 2, issue 5, with the name "landing page."\

# **When first submitting your pull request, make sure to assign it on github to your team members who will be responsible for validating your pull request and make sure there are no merge conflicts.**

Assignees

# **After assigning to a particular member set, we can then assign it to a proper project if you are working across multiple repos. This is especially important for the above reason, and is a fantastic way to organize your pull requests without mixing up where they are intended to go.**

Projects

# **We can also assign a label to the pull request. This lets your PM/PO/ etc know what this pull request applies to. This makes assessing what we're looking for very quick and easy as a Project Manager, or other leader within the team.**

Labels

# **When submitting your pull request, be descriptive in both your topic name (IntelliJ Configuration(s)) and also assign it to a numbered issue. In this case, we've assigned this #12, because at the time of creation, there were 11 issues, and this was the logical next number to assign it to. 

PullComment

# **Ensure you include details. As <insert role> I did the following: <insert steps, include images if applicable>**

# **When submitting a comment for a pull request, please label your pull request as a solution to another issue.**

For example, reference the below image, and what I have put into the comment that will resolve our number 12 issue of no .idea file that's ignored in our .gitignore file.

ResolvesComment

# **We also want to include unit tests if and when possible to make sure we're getting full coverage of our code. This will simulate a real world environment where tests are extremely important.**

[Unit Test Example](https://www.geeksforgeeks.org/unit-testing-software-testing/)

### CODE REVIEW

File: app/src/main/java/com/example/nsc_events/screen/RegisterationPage.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            expanded = selectedRole.isNotEmpty(),
+            onDismissRequest = { },
+        ) {
+            DropdownMenuItem(onClick = { selectedRole = "Student" }) {
+                Text("Student")
+            }
+            DropdownMenuItem(onClick = { selectedRole = "Faculty" }) {
+                Text("Faculty")
+            }
+        }
```
Comment (taylorpapke) [inline app/src/main/java/com/example/nsc_events/screen/RegisterationPage.kt:59]: I'm getting the error No value passed for parameter 'text' for line 53 when I try to run the code.

Comment (Robel-003) [reply app/src/main/java/com/example/nsc_events/screen/RegisterationPage.kt:57]: I'm also getting that same error. It's saying there a type mismatch when I hover over the "Text(item)"

Type mismatch.
Required: MutableInteractionSource
Found: () → Unit

**Review by kaimanasse15 — COMMENTED** [review]
How can I fix this problem?

**Comment by Robel-003** [comment]
> How can I fix this problem?

I'll research and see if I can find some resources for a solution. In the meantime I'd recommend scheduling a time to meet with one of the lead developers for assistance

**Comment by taylorpapke** [comment]
@theGaryLarson @tinpham5614

**Comment by taylorpapke** [comment]
@kaimanasse15 
- let's use the example already available for the registration (sign-up) page on belindas-android repo @ https://github.com/SeattleColleges/belindas-closet-android/blob/main/app/src/main/java/com/example/belindas_closet/screen/SignUp.kt

- lets also change the name of this PR and link it to issue https://github.com/SeattleColleges/nsc-events-android/issues/46

- we need to follow the branch and PR naming conventions located at the top of our wiki: https://github.com/SeattleColleges/nsc-events-android/wiki

**Comment by taylorpapke** [comment]
@kaimanasse15 I was able to get this running locally and everything seems to be behaving fine. Ideally, it would be nice to have the fields be as identical as possible to the ones in Belindas-closet-android:
sign-up-page
The code for that is here:
https://github.com/SeattleColleges/belindas-closet-android/blob/main/app/src/main/java/com/example/belindas_closet/screen/SignUp.kt

**Review by taylorpapke — CHANGES_REQUESTED** [review]
- Change the fields to match the ones in the screenshot from Belindas-closet-android.
- resolve merge conflicts.

**Comment by taylorpapke** [comment]
@kaimanasse15 helpful doc on resolving merge conflicts: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

**Review by taylorpapke — DISMISSED** [review]

**Review by Robel-003 — DISMISSED** [review]
This is compiling and running. Just like Taylor mentioned, we want the fields to be similar to that of Belinda's Closet registration page, (First Name, Last Name, Email, Password, Confirm Password)
 
Also this page is missing a back button to the Login page.

**Comment by taylorpapke** [comment]
Hey @kaimanasse15, looks like the lint check failed for the following lines:
Screenshot 2023-11-09 at 8 41 19 PM

**Comment by bcko** [comment]
@Ali-7s said that the sign up page is already implemented and merged on main branch. Before we approve and merge this pull request, @taylorpapke @Robel-003 Let's make sure this pull request is not duplicating the effort.

**Comment by Ali-7s** [comment]
> @Ali-7s said that the sign up page is already implemented and merged on main branch. Before we approve and merge this pull request, @taylorpapke @Robel-003 Let's make sure this pull request is not duplicating the effort.

I worked on it here: https://github.com/SeattleColleges/nsc-events-android/pull/87

**Comment by taylorpapke** [comment]
> > @Ali-7s said that the sign up page is already implemented and merged on main branch. Before we approve and merge this pull request, @taylorpapke @Robel-003 Let's make sure this pull request is not duplicating the effort.
> 
> I worked on it here: #87

Yes, this is unfortunately a duplicate. Sorry @kaimanasse15 for not seeing this sooner. I will close this and you can continue working on your other assigned user story.

**Pull request closed without merging**